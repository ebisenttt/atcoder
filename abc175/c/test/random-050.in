507595417301091 647687321255800 198365599808553
