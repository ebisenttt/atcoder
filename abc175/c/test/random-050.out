110864217683985
